safe_transaction_service/history/views.py

```diff
@@ -167,7 +167,7 @@ class IndexingView(GenericAPIView):
     serializer_class = serializers.IndexingStatusSerializer
     pagination_class = None  # Don't show limit/offset in swagger
 
-    @method_decorator(cache_page(5 * 60))  # 5 minutes
+    @method_decorator(cache_page(15))  # 15 seconds
     def get(self, request):
```

Comment:
I'd be defensive on this, 15 seconds seems to low, maybe 2 or 3 minutes?

Reply:
Why? In this case database queries are very fast and it gives us more information about what's going on on the service